curated_transformers/tokenization/hf_loader.py

```diff
@@ -51,10 +52,20 @@ def _convert_encoder(
         return _convert_sentencepiece_encoder(model, tokenizer)  # type: ignore
     elif isinstance(tokenizer, transformers.BertJapaneseTokenizer):
         return _convert_bert_japanese_encoder(model, tokenizer)
-
-    raise ValueError(
-        f"Loading from the '{type(tokenizer)}' huggingface tokenizer is not supported"
-    )
+    else:
+        supported_tokenizers = (
+            transformers.BertTokenizerFast,
+            transformers.RobertaTokenizerFast,
+            transformers.XLMRobertaTokenizerFast,
+            transformers.CamembertTokenizerFast,
+            transformers.BertJapaneseTokenizer,
```

Comment:
Would be nice to have this as a global 'constant' near the top of the file. Makes it easier to maintain/inspect.

Reply:
Done.